backend/plugin/db/mysql/mysql.go

```diff
@@ -166,6 +166,10 @@ func (driver *Driver) Execute(ctx context.Context, statement string, _ bool, opt
 	if err != nil {
 		return 0, errors.Wrapf(err, "failed to deal with delimiter")
 	}
+
+	if driver.dbType == storepb.Engine_TIDB {
```

Comment:
Move above the first block or remove the dealWithDelimeter in TiDBExecute.

Reply:
Done.